src/renderer/components/dex/MarketSelection.vue

```diff
@@ -35,13 +35,16 @@
 
       <div class="token-details">
         <aph-token-icon :symbol="$store.state.currentMarket.quoteCurrency"></aph-token-icon>
-        <div class="market-name">{{ $store.state.currentMarket.marketName }}</div>
-        <div class="price">
-          <span class="label">{{$t('priceUc')}}</span>
-          <div class="base-price">
-            {{ $formatTokenAmount($store.state.tradeHistory.close24Hour) }}
-          </div>
-          <div :class="['change', {decrease: $store.state.tradeHistory.change24Hour < 0, increase: $store.state.tradeHistory.change24Hour > 0}]">{{ $formatNumber($store.state.tradeHistory.change24Hour) }}</div>
+        <div class="base-price">
+          {{ $formatTokenAmount($store.state.tradeHistory.close24Hour) }}
+        </div>
+        <div class="base-price-converted">
+          {{ $formatMoney($store.state.tradeHistory.close24Hour * baseCurrencyUnitPrice) }}
+        </div>
+        <span class="label">{{$t('change')}} ({{$store.state.currentMarket.quoteCurrency}})</span>
```

Comment:
Please add spaces between curly braces and their expression.

```vue
<span class="label">{{ $t('change') }} ({{ $store.state.currentMarket.quoteCurrency }})</span>
```

Reply:
Nice catch. I wonder if there is linter rule we can add for this.